README.markdown

```diff
@@ -13,7 +13,7 @@ resource health.
 
 ## Who is Pacemaker?
 
-Pacemaker is distributed by [ClusterLabs](http://www.clusterlabs.org).
+Pacemaker is distributed by [ClusterLabs](https://www.clusterlabs.org/).
```

Comment:
While you're at it, there's a bunch of other references to "http://www.clusterlabs.org" - all the conf.py files in doc/sphinx, several in doc/crm_fencing.html, and one in extra/cluster-init.

Reply:
Good catch, replaced last commit

The conf.py are created from conf.py.in, and crm_fencing.html is created from crm_fencing.txt. crm_fencing.html and cluster-init are woefully outdated overall, but that will have to wait for another time ...